### ISSUE TITLE
#2127001636------------- Elements must have sufficient color contrast

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#591]
<strong>Source:</strong>`<div class="i-graphicstat-info-label">People</div>`
<strong>Element Location: </strong>.sc-bhVIhj.bJVytO:nth-child(2) > .i-graphicstat > .i-graphicstat-info-label
<strong>Impact: </strong>serious
<strong>failureSummary:</strong> Fix any of the following:
  Element has insufficient color contrast of 3.19 (foreground color: #909090, background color: #ffffff, font size: 8.3pt (11.104px), font weight: normal). Expected contrast ratio of 4.5:1
Tags: cat.color,wcag2aa,wcag143,ACT
Screenshot:`C:\Users\khy12817\Desktop\Test.PNG`
